src/protocol/sort/reshare.rs

```diff
@@ -79,26 +88,53 @@ impl<F: Field> Reshare<F> {
     }
 }
 
+/// For malicious reshare, we run semi honest reshare protocol twice, once for x and another for rx and return the results
```

Comment:
I think that this code has already landed. Can you rebase?

Reply:
Yes let me do that